src/sfctl/tests/help_text_test.py

```diff
@@ -0,0 +1,364 @@
+            # returned_string and err are returned as bytes
+            (returned_string, err) = pipe.communicate()
+
+            # If there is an error message, then assert
+            if err:
+                err = err.decode('utf-8')
+                print(err, file=stderr)
+                self.assertEqual(b'', err, msg='ERROR: in command: ' + help_command)
+
+            # If there is no help text, then assert
+            if not returned_string:
+                self.fail('No help text in command: ' + help_command)
+
+            returned_string = returned_string.decode('utf-8')
+            lines = returned_string.splitlines()
+
+            for line in lines:
+
+                print(line, file=stderr)
+
+                # Skip empty lines
+                if not line.strip():
+                    continue
+
+                # Check if we want to mark the start of a new section
+                # Check this by seeing if the line is a top level description, ie: 'Commands:'
+                # These are characterized by a new line with text starting without spaces
+                read_section_output = self._validate_output_read_section_name(line)
+                if read_section_output is not None:
+                    section = read_section_output
+                    continue
+
+                    # If this line is a section designator, no additional processing
+                    # is required. Move on to the next line.
+
+                updated_indices = self._validate_output_validate_line(command_input, line, section,
+                                                                      subgroups, commands,
+                                                                      subgroups_index,
+                                                                      commands_index)
+                subgroups_index = updated_indices[0]
+                commands_index = updated_indices[1]
+
+            # If section is still 'Start', the something has gone wrong
+            self.assertNotEqual('Start', section,
+                                msg='Command {0} has incomplete help text: {1}'.format(help_command, returned_string)) #pylint: disable=line-too-long
+
+            # Check that we have traversed completely through both
+            # subgroups and commands
+            self.assertEqual(len(commands), commands_index,
+                             msg=('Not all commands listed in help text for '
+                                  + help_command
+                                  + '. This may be a problem due incorrect expected ordering.'))
+            self.assertEqual(len(subgroups), subgroups_index,
+                             msg=('Not all subgroups listed in help text for '
+                                  + help_command
+                                  + '. This may be a problem due incorrect expected ordering.'))
+
+            print(file=stderr)
+
+        except Exception: # pylint: disable=broad-except
```

Comment:
Don't disable `broad-except`. Tests will fail anyways if an exception is raised.

Reply:
What do you mean by this comment? Are you saying that I should not have this at all? This provides a clearer picture of what the error is I feel like, rather than just having the error message. It organizes it better